cmd/main.go

```diff
@@ -53,8 +53,9 @@ var (
 	resourceTagsStr                 = flag.String("resource-tags", "", "Resource tags to attach to each volume created. It is a comma separated list of tags of the form '<parentID_1>/<tagKey_1>/<tagValue_1>...<parentID_N>/<tagKey_N>/<tagValue_N>' where, parentID is the ID of Organization or Project resource where tag key and value resources exist, tagKey is the shortName of the tag key resource, tagValue is the shortName of the tag value resource. See https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing for more details.")
 
 	// Feature lock release specific parameters, only take effect when feature-lock-release is set to true.
-	featureLockRelease = flag.Bool("feature-lock-release", false, "if set to true, the node driver will support Filestore lock release.")
-
+	featureLockRelease    = flag.Bool("feature-lock-release", false, "if set to true, the node driver will support Filestore lock release.")
+	featureLockReleaseV2  = flag.Bool("feature-lock-release-v2", false, "if set to true, the node driver will not support Filestore lock release.")
```

Comment:
"... not support Filestore v1 lock release"

Reply:
also looking at the implementation if both set to true, the v2 implementation take precedence. we should clearly call that out in the flag comment

Reply:
done